e2e/pages/respondToClaim/uploadResponseDocument.page.js

```diff
@@ -0,0 +1,19 @@
+const { I } = inject();
+
+module.exports = {
+
+  fields: {
+    respondentResponseDocument: {
+      id: '#respondentResponseDocument_responseDocument'
+    }
+  },
+
+  async uploadResponseDocuments (file) {
+    I.waitForElement(this.fields.respondentResponseDocument.id);
+    await I.attachFile(this.fields.respondentResponseDocument.id, file);
+    await I.waitForInvisible(locate('.error-message').withText('Uploading...'));
```

Comment:
would be nice to have an upload document common fragment method

Reply:
will pass that